shared/lib_battery_dispatch.h

```diff
@@ -57,7 +57,7 @@ class dispatch_t
 public:
 
 	enum FOM_MODES { FOM_AUTOMATED_ECONOMIC, FOM_PV_SMOOTHING, FOM_CUSTOM_DISPATCH, FOM_MANUAL };
-	enum BTM_MODES { PEAK_SHAVING, MAINTAIN_TARGET, CUSTOM_DISPATCH, MANUAL, FORECAST };
+	enum BTM_MODES { PEAK_SHAVING, MAINTAIN_TARGET, CUSTOM_DISPATCH, MANUAL, FORECAST, CARBON_FREE };
```

Comment:
I hate to slow things down, but having it be CARBON_FREE in the code and self consumption in the UI is going to set us up for confusion later. Mind fixing it to match the UI?

Reply:
Ha, no you're absolutely right. I had that on my list, will change now.